src/CreatesMenuItems.php

```diff
@@ -93,11 +93,33 @@ public function &add(string $title, $config = null)
 
             return $this;
         }
+        
+
+		// Is it another Menu?
+		if(starts_with($config, 'menu:') || starts_with($config, 'sub-menu:')) {
+			$slug = $item['__slug__'] = snake_case($title);
+
+			if(starts_with($config, 'menu:')) {
+				$items =& $this->_items;
+				$menu = app('menu')->get(str_replace_first('menu:', '', $config));
```

Comment:
Should either return $this (soft failure) or throw exception if menu doesn't exist.